src/Engine/include/OpenLoco/Engine/Ui/Size.hpp

```diff
@@ -15,4 +15,17 @@ namespace OpenLoco::Ui
         {
         }
     };
+
+    // TODO: Remove the Size struct and rename this one to Size once interop is gone.
+    struct Size32
+    {
+        int32_t width = 0;
+        int32_t height = 0;
```

Comment:
Why do we need 32-bit values for the dimensions? I can't imagine windows to exceed 65k on either axis tbh.

Reply:
Because of integer promotion, int16_t(1) + 1 will result in int so its much easier to just use ints in this case.

Reply:
To add some context, for some reason when switching to makeWidgets it complained about narrowing the type via cast, so if I did the casts it would have become one massive ugly mess. I'm actually not sure to why this has worked before without issues, we do suppress a lot of warnings.

Reply:
Ahh, integer promotion... I'd prefer using smaller types when possible, but I can see why this is definitely less hassle. Not worth the byte-pinching, indeed.